cs.html

```diff
@@ -554,6 +695,102 @@ <h2>Dictionary BidirectionalStreamEventInit Members</h2>
+                    <td class="prmNullFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmOptFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmDesc"></td>
+                  </tr>
+                  <tr>
+                    <td class="prmName">eventInitDict</td>
+                    <td class="prmType"><code><a>DatagramsReceivedEventInit</a></code></td>
+                    <td class="prmNullFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmOptFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmDesc"></td>
+                  </tr>
+                </tbody>
+              </table>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="DatagramsReceivedEvent" data-dfn-for="DatagramsReceivedEvent"
+          class="attributes">
+            <dt><code>data</code> of type <span class=
+            "idlAttrType"><a>Uint8Array</a></span>, readonly</dt>
+            <dd>
+              <p>The <dfn id="dom-datagramsreceivedevent-data"><code>data</code></dfn>
+              attribute represents the <code><a>data</a></code> received. </p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+      <div>
+          <p>The <dfn><code>DatagramsReceivedEventInit</code></dfn> dictionary includes
+          information to construct a DatagramsReceivedEvent .</p>
+        <pre class="idl">dictionary DatagramsReceivedEventInit : EventInit {
+             Uint8Array data;
+};</pre>
+        <section>
+          <h2>Dictionary DatagramsReceivedEventInit Members</h2>
+          <dl data-link-for="DatagramsReceivedEventInit" data-dfn-for=
+          "DatagramsReceivedEventInit" class="dictionary-members">
+            <dt><dfn><code>data</code></dfn> of type <span class=
+            "idlMemberType"><a>sequence&ltUint8Array?&gt</a></span></dt>
+            <dd>
+              <p>The payloads of the datagrams received, with possible null
+              values. </p>
+              <p>A null value indicates that the user agent dropped 1 or more
+              datagrams after receiving and acking them because the event
+              handler was not able to handle incoming datagrams quickly enough
+              and the user agent has decided that dropping them is
+              better than buffering them.  How much to buffer before dropping is
+              left to the implementation of the user agent to decide, but
+              implementations should seek to minimize buffering and assume that
+              event handlers will be able to handle dropped datagrams.</p>
+              <p> The event provides a sequence of datagrams (rather than a
```

Comment:
Should this text about the sequence of datagrams be moved above in the DatagramsReceivedEvent text after "event uses the DatagramsReceivedEvent interface."?

Reply:
Yeah, whoops.  Let me fix that.